install/install/includes/sql.sql

```diff
@@ -797,14 +797,14 @@ CREATE TABLE us_ip_blacklist (
 CREATE TABLE us_ip_list (
   id SERIAL PRIMARY KEY,
   ip VARCHAR(50) NOT NULL,
-  user_id INT NOT NULL,
-  timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+  user_id INTEGER NOT NULL,
+  "timestamp" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
 CREATE OR REPLACE FUNCTION update_timestamp()
```

Comment:
Is this procedure used anywhere?

Reply:
What do you mean? Is it not correct?

Reply:
You talking about double quotes?

Reply:
No, we are talking about the `update_timestamp()` function, which is not present in the original `sql.sql` AFAICS.

Reply:
So its a trigger, not a procedure, and since the original query was ``timestamp` timestamp NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp()` the `ON UPDATE` method does not exist in pgsql. So I searched Google and found this alternative.

Reply:
[StackOverFlow](https://stackoverflow.com/questions/1035980/update-timestamp-when-row-is-updated-in-postgresql)

Reply:
Okay then, please remove the trigger and use the built-in `CURRENT_TIMESTAMP` function as you used in other places.

Reply:
Ok professor